db/schema/01_schema.sql

```diff
@@ -0,0 +1,39 @@
+DROP TABLE IF EXISTS users CASCADE;
+DROP TABLE IF EXISTS items CASCADE;
+DROP TABLE IF EXISTS orders CASCADE;
+DROP TABLE IF EXISTS order_items CASCADE;
+
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY NOT NULL,
+  first_name VARCHAR(255) NOT NULL,
+  last_name VARCHAR(255) NOT NULL,
+  phone_number VARCHAR(255) NOT NULL,
+  email VARCHAR(255) NOT NULL,
+  password VARCHAR(255) NOT NULL,
+  isOwner BOOLEAN DEFAULT FALSE
+);
+
+CREATE TABLE items (
+  id SERIAL PRIMARY KEY NOT NULL,
+  name VARCHAR(255) NOT NULL,
+  description TEXT,
+  price INT NOT NULL,
+  image_url TEXT
+);
```

Comment:
I'm thinking maybe can add one "category" column in items table so the items in the menu can get categorized? Could be like "Bubble Tea", "Fruit Tea", "Tea Latte", "Slush", etc.